tests/baselines/reference/classExpression4.types

```diff
@@ -1,22 +1,22 @@
 === tests/cases/conformance/classes/classExpressions/classExpression4.ts ===
 let C = class {
->C : typeof (Anonymous class)
+>C : any

```

Comment:
It becomes any because C is accessed again as part of method resolution of class expression.